package.json

```diff
@@ -80,9 +80,7 @@
     "@types/node-schedule": "^2.1.7",
     "@types/passport": "^1.0.11",
     "@types/passport-local": "^1.0.35",
-    "@types/passport-saml": "^1.1.3",
```

Comment:
this is still used, but it shouldnt be in both dependencies and dev dependencies (same with a lot of these but this is the only one i tracked down)